test/common.js

```diff
@@ -142,7 +147,20 @@ exports.validateContent = function validateContent(data, t, options) {
   }
 
   // Check report System Information section
-  const sysInfoSection = getSection(reportContents, 'System Information');
+  // If the report contains a glibc version, check it against libc.so.6
+  const glibcMatch = glibcRE.exec(nodeReportSection);
+  if (glibcMatch != null) {
+    const libcMatch = libcRE.exec(sysInfoSection);
+    if (libcMatch != null) {
+      const libcPath = libcMatch[1];
+      const child = child_process.spawnSync(libcPath, { encoding: 'utf8' });
```

Comment:
The official [FAQ says](https://sourceware.org/glibc/wiki/FAQ#How_can_I_find_out_which_version_of_glibc_I_am_using_in_the_moment.3F): 
> If you want to find out about the version from the command line simply run the libc binary. This is probably not possible on all platforms but where it is simply locate the libc shared library and start it as an application. 

I've changed the test to use `strings`.